projects/aca-content/src/lib/components/view-profile/view-profile.component.html

```diff
@@ -26,17 +26,17 @@ <h3 class="app-profile">{{'APP.EDIT_PROFILE.MY_PROFILE' | translate}}</h3>
       <div class="app-profile-general-section">
         <button
           mat-icon-button
-          (click)="toggleGeneralDropdown()"
-          [attr.title]="`APP.TOOLTIPS.${generalSectionDropdown ? 'COLLAPSE' : 'EXPAND'}_SECTION` | translate"
-          [attr.aria-label]="`APP.TOOLTIPS.${generalSectionDropdown ? 'COLLAPSE' : 'EXPAND'}_SECTION` | translate">
-          <mat-icon>{{ generalSectionDropdown ?  'expand_more' : 'chevron_right'}}</mat-icon>
+          (click)="toggleGeneralSection()"
+          [attr.title]="`APP.TOOLTIPS.${generalSectionExpanded ? 'COLLAPSE' : 'EXPAND'}_SECTION` | translate"
```

Comment:
maybe this can be a getter in the typescript class, without evaluating at the template side 2 times. Also the aria-label seems to be the same tooltip, I guess can be omitted